### ISSUE TITLE
Package has no exec file

### ISSUE THREAD

**Issue by SWAROOPASHIGLI** [NxRLab/ModernRobotics#31]
The installed book software has no executable files

**Comment by HuanWeng** [comment]
Right. There're only functions in the software. Please check the documentation for more information.
